README.md

```diff
@@ -389,6 +418,12 @@ recovery code is the new recovery code you must show to the user from him to sav
 transaction.recover({ recoveryCode: recoveryCode });
 ```
 
+#### transaction.serialize() 
+The .serialize() method creates a plain javascript Object that should remind opaque 
```

Comment:
...I'm not sure what is meant here, with the object being opaque to the end user.  In all examples, the object is not sent to the end user.